src/Operators/laplacian_operators.jl

```diff
@@ -2,19 +2,19 @@
 ##### Horizontal Laplacians
 #####
 
-@inline function ∇²hᶜᶜᵃ(i, j, k, grid, c)
-    return 1/Vᵃᵃᶜ(i, j, k, grid) * (δxᶜᵃᵃ(i, j, k, grid, Ax_∂xᶠᵃᵃ, c) +
-                                    δyᵃᶜᵃ(i, j, k, grid, Ay_∂yᵃᶠᵃ, c))
+@inline function ∇²hᶜᶜᶜ(i, j, k, grid, c)
```

Comment:
The changes look good to me and I don't have any problems but I do have a question.  There are a lot of different operators and volumes because of the different superscripts, and I don't pretend to know what's all there.  But I gather that if we have `V^aac` that will compute hte volumes for any chase in the first two indices, but the third must be center.  Is there just one way of writing the above or are there various options?  I'm just curious to better understand how to deal with operators in the future.

Reply:
I think ultimately we would like all of our operators to be fully general for three-dimensionally curvilinear grid. At the moment we are currently transitioning from code that works only when the vertical direction is stretched to code that works when all three directions are stretched AND the grid is _horizontally_ curvilinear. We are taking the approach that we will not write operators that are "more general" than what we can explicitly write tests for (we don't have three-dimensionally curvilinear tests, so we can't write fully general three-dimensionally curvilinear operators). The downside of this approach is that the code is messier; there is currently a lot of code that will not run (hopefully it will throw `Method not found` error) when using a horizontally-curvilinear grid. The upside, we hope, is that there are fewer silent and deadly bugs.

Reply:
An operator that looks like `V^aac` is therefore only valid for a grid that is rectilinear, horizontally regular, and _at most_ vertically stretched.